api/src/opentrons/execute.py

```diff
@@ -313,18 +313,18 @@ def execute(
 
     :param protocol_file: The protocol file to execute
     :param protocol_name: The name of the protocol file. This is required
-                          internally, but it may not be a thing we can get
-                          from the protocol_file argument.
+        internally, but it may not be a thing we can get
+        from the protocol_file argument.
```

Comment:
```suggestion
        from the ``protocol_file`` argument.
```
Mono for an argument name.